class.frame-nonce-preview.php

```diff
@@ -95,6 +101,19 @@ public function set_post_to_publish( $posts ) {
 
 		return $posts;
 	}
+
+	/**
+	 * Handle validation for autosave preview request
+         *
```

Comment:
Thanks... I don't have a proper jetpack dev setup so I was mangling this with nano. Didn't notice the spacing.